tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist=True
-envlist = py35
+envlist = py310
```

Comment:
Why are we changing this to py310? Since the charm supports xenial, it would be better to leave it at py35.